From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Victor Fuentes <vmfuentes64@gmail.com>
Date: Thu, 1 Aug 2024 15:53:16 -0400
Subject: [PATCH] Modifies the users module to only set passwords of user and
 root

as the users will have already been created in the configuration.nix
file
---
 src/modules/users/Config.cpp | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/src/modules/users/Config.cpp b/src/modules/users/Config.cpp
index cd56bc3e2..9b09b36cd 100644
--- a/src/modules/users/Config.cpp
+++ b/src/modules/users/Config.cpp
@@ -1028,12 +1028,6 @@ Config::createJobs() const
 
     Calamares::Job* j;
 
-    if ( !m_sudoersGroup.isEmpty() )
-    {
-        j = new SetupSudoJob( m_sudoersGroup, m_sudoStyle );
-        jobs.append( Calamares::job_ptr( j ) );
-    }
-
     if ( getActiveDirectoryUsed() )
     {
         j = new ActiveDirectoryJob( m_activeDirectoryAdminUsername,
@@ -1043,20 +1037,11 @@ Config::createJobs() const
         jobs.append( Calamares::job_ptr( j ) );
     }
 
-    j = new SetupGroupsJob( this );
-    jobs.append( Calamares::job_ptr( j ) );
-
-    j = new CreateUserJob( this );
-    jobs.append( Calamares::job_ptr( j ) );
-
     j = new SetPasswordJob( loginName(), userPassword() );
     jobs.append( Calamares::job_ptr( j ) );
 
     j = new SetPasswordJob( "root", rootPassword() );
     jobs.append( Calamares::job_ptr( j ) );
 
-    j = new SetHostNameJob( this );
-    jobs.append( Calamares::job_ptr( j ) );
-
     return jobs;
 }
